src/main/java/org/assertj/vavr/api/SeqShouldBeSorted.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
+ */
+package org.assertj.vavr.api;
+
+import io.vavr.collection.Seq;
+import org.assertj.core.error.BasicErrorMessageFactory;
+import org.assertj.core.error.ErrorMessageFactory;
+
+import java.util.Comparator;
+
+/**
+ * Creates an error message indicating that an assertion that verifies a sequence of elements is sorted failed.
+ *
+ * @author Michał Chmielarz
+ */
+class SeqShouldBeSorted extends BasicErrorMessageFactory {
```

Comment:
Aren't those covered by IterableAssert from assertj-core?

Reply:
No, `IterableAssert` is about non-sequential collections while `SeqAssert` is about sorted sequences.